.env

```diff
@@ -0,0 +1,7 @@
+MYSQL_HOST=localhost
```

Comment:
It is not a good idea to push files with credentials (DB password, etc) to github. Especially to public repo.
Instead you can push .env.example with the required properties and empty values. 

I see you have the the .env in .gitignore
so you can check this https://stackoverflow.com/questions/3833561/why-doesnt-git-ignore-my-specified-file